scripts/jina-hub-update.py

```diff
@@ -2,97 +2,281 @@
 [encoders, crafters, indexers, rankers, evaluators, classifiers etc.].
 It also adds the required jina version.
 Commits the change in the branch and raises a PR for the executor.
+Then attempts to automatically merge the PRs
 """
 import glob
 import os
+import sys
+import time
+import traceback
+from typing import List, Optional, Tuple
 
 import git
+import requests
 import semver
-from github import Github
+from github import Github, PullRequestMergeStatus, Repository
+from github.Issue import Issue
+from github.PullRequest import PullRequest
 from ruamel.yaml import YAML
 
+WAIT_BETWEEN_PR_CHECKS = 5 * 60
+FIX_MODULE_TEMPLATE = 'fix module '
+TAG_IN_ISSUES = os.environ.get('TAG_IN_ISSUES', '')
+MODULES_REPO = os.environ.get('MODULES_REPO')
+if MODULES_REPO is None:
+    print(f'Error: MODULES_REPO needs to be set. Exiting...')
+    sys.exit(1)
+
 # this one has PR push access
 g = Github(os.environ["GITHUB_TOKEN"])
 
 yaml = YAML()
 
 
+def create_pr(fpath, module, jina_core_version, hub_repo, hub_origin, gh_hub_repo) -> Optional[PullRequest]:
+    """for each module with manifest.yml attempts to bump version and add jina core version.
+    then opens PR"""
+    pr = None
+    print(f'handling {module}...')
+    with open(fpath) as fp:
+        info = yaml.load(fp)
+        # make sure the (possibly) existing version is older
+        if 'jina_version' in info.keys():
+            # TODO should this be list?
```

Comment:
I think this TODO is now answered right? It should not?

Reply:
Yep, removed